.github/workflows/sync-with-upstream.yaml

```diff
@@ -50,33 +50,41 @@ jobs:
           && cat repo/.github/workflows/.template_version || echo "v0.0.0")
         echo "Latest version is $LATEST_VERSION and current version is $CURRENT_VERSION."
 
-        if [ $CURRENT_VERSION == $LATEST_VERSION ]; then
+        # Check if the template repo was changed/updated.
+        if [ "$CURRENT_VERSION" == "$LATEST_VERSION" ]; then
           echo "Unable to find a new version, exiting..."
-        else
```

Comment:
Can you please keep the `else` to have an explicit `if ... else ...`  pattern?

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/537/commits/0f96f6eaa30a9dcab5f3d2e23bf0024160aed280.